pallets/farming/src/weights.rs

```diff
@@ -63,6 +63,7 @@ pub trait WeightInfo {
 	fn refresh_pool(a: u32, ) -> Weight;
 	fn prepare_accounts_for_vesting(a: u32, b: u32, ) -> Weight;
 	fn vest_account_rewards(a: u32, ) -> Weight;
+	fn set_lp_min_xor_for_bonus_reward() -> Weight;
```

Comment:
I don't see the benchmarks. You need to add the benchmark of the extrinsic in the `benchmarking.rs` and modiy the `WeightInfo` inside the `farming::Config` in the `runtime\src\lib.rs`